lightcycle-lib/src/main/java/com/soundcloud/lightcycle/SupportFragmentLightCycleDispatcher.java

```diff
@@ -14,10 +15,17 @@
 
 public class SupportFragmentLightCycleDispatcher<T extends Fragment>
         implements LightCycleDispatcher<SupportFragmentLightCycle<T>>, SupportFragmentLightCycle<T> {
+
     private final Set<SupportFragmentLightCycle<T>> fragmentLightCycles;
+    private final LightCycleBinderHelper binderHelper;
 
     public SupportFragmentLightCycleDispatcher() {
+        this(new LightCycleBinderHelper.Factory());
+    }
+
+    public SupportFragmentLightCycleDispatcher(LightCycleBinderHelper.Factory factory) {
```

Comment:
Do we want to expose a public constructor for the Dispatchers with custom BinderHelpers?